pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "hyperleaup"
-version = "0.1.1"
+version = "0.1.2"
 authors = [
     { name="Will Girten", email="will.girten@databricks.com" },
```

Comment:
@dustinvannoy-db mind if we add you as an author here?

Reply:
Sounds good to me.